src/cli.js

```diff
@@ -58,6 +58,19 @@ export function getConfig({useCLI=true} = {}) {
       type: 'boolean',
       default: false,
     })
+    .option('scan-file', {
+      alias: ['f'],
+      describe: 'Include only a single files in the scan',
+      type: 'string',
+      requiresArg: true,
+      coerce: (value) => {
```

Comment:
Huh, why is this here? Any harm in supplying it multiple times? (It's weird but harmless I'd think; I don't think we do checks like this elsewhere. If we should start, we should do this for anything that should only be supplied once.)